src/modules/workspaces/workspaces.usecase.ts

```diff
@@ -419,6 +419,72 @@ export class WorkspacesUsecases {
     await this.workspaceRepository.deleteInviteBy({ id: invite.id });
   }
 
+  async getUserInvites(user: User, limit: number, offset: number) {
+    const invites = await this.workspaceRepository.findInvitesBy(
+      {
+        invitedUser: user.uuid,
+      },
+      limit,
+      offset,
+    );
+
+    const invitesWithWorkspaceData = await Promise.all(
+      invites.map(async (invite) => {
+        const workspace = await this.workspaceRepository.findById(
+          invite.workspaceId,
+        );
+
+        return {
+          ...invite,
+          workspace: workspace.toJSON(),
+        };
+      }),
+    );
+
+    return invitesWithWorkspaceData;
+  }
+
+  async getWorkspacePendingInvitations(
+    workspaceId: WorkspaceAttributes['id'],
+    limit: number,
+    offset: number,
+  ) {
+    const invites = await this.workspaceRepository.findInvitesBy(
+      {
+        workspaceId,
+      },
+      limit,
+      offset,
+    );
+
+    const sharedWithUuids = invites.map((invite) => invite.invitedUser);
```

Comment:
Why `sharedWithUuids`? Maybe it's because you forgot to rename? If it contains the uuids of the invited users it looks better `invitedUsersUuuids` to me